docs/blueprints/make_blueprints_table.py

```diff
@@ -0,0 +1,65 @@
+import re
+import os
+import glob
+
+def create_toc_tree():
+    main_title = '\n' + '#' * 10 + '\nBlueprints\n' + '#' * 10 + '\n'
+    sub_title = '\nTable of Blueprints\n\n'
+    
+    file_header = [':orphan:\n']
+    file_header.append(main_title)
+    file_header.append(sub_title)
+    
+    toc_tree_text = ['.. toctree::\n', '  :glob:\n', '  :hidden:\n','\n', '  EP*\n\n',]
+    
+    file_header = ''.join(file_header)
+    toc_tree_text = ''.join(toc_tree_text)
+    
+    return file_header, toc_tree_text
+    
+    
+def create_table(directory):
+    #Create the table header
+    table_header = ''
+    table_header += '+' + '-' * 23 + '+' + '-' * 82 + '+' + '-' * 20 + '+\n'
+    table_header += '|' + ' Blueprint'  + ' ' * 13 + '|' + ' Title' + ' ' * 76 + '|' + ' Status' + ' ' * 13 + '|\n'
+    table_header += '+' + '=' * 23 + '+' + '=' * 82 + '+' + '=' * 20 + '+\n'
```

Comment:
Please use a global string for this, disable the lint 80-char check if necessary.